src/resources/js/components/AccountEdit.vue

```diff
@@ -0,0 +1,181 @@
+<template>
+  <v-container>
+
+  <v-alert
+    v-model="message.show"
+    :class="message.type"
+    dismissible
+    >
+      {{ message.content }}
+    </v-alert>
+
+  <h2>Mon Compte</h2>
+
+  <v-form
+    ref="form"
+    v-model="valid"
+    lazy-validation
+  >
+    <v-text-field
+      v-model="updateUserInfo.fname"
+      :counter="10"
+      :rules="nameRules"
+      label="Prenom"
+    />
+
+    <v-text-field
+      v-model="updateUserInfo.lname"
+      :counter="10"
+      :rules="nameRules"
+      label="Nom"
+    />
+
+    <v-text-field
+      v-model="updateUserInfo.email"
+      :rules="emailRules"
+      label="E-mail"
+    />
+
+    <v-text-field
+      v-model="updateUserInfo.password"
+      :append-icon="showPassword ? 'mdi-eye' : 'mdi-eye-off'"
+      :type="showPassword ? 'text' : 'password'"
+      label="Mot de passe"
+      value=""
+      class="input-group--focused"
+      @click:append="showPassword = !showPassword"
+    />
+
+    <v-btn
+      :disabled="!valid"
+      color="success"
+      class="mr-4"
+      @click="submitChange"
+    >
+      Appliquer
+    </v-btn>
+
+  </v-form>
+
+  </v-container>
+</template>
+
+<script>
+export default {
+  name: 'AccountEdition',
+  data() {
+    return {
+      valid: true, // if a rules is not fully satisfied this will disable the submit button
+      nameRules: [
+        (v) => v === '' || v.length <= 10 || 'Le nom doit contenir moins de 10 caractères',
+      ],
+      emailRules: [
+        (v) => v === '' || /^\w+([.-]?\w+)*@\w+([.-]?\w+)*(\.\w{2,})+$/.test(v) || 'l\'E-mail doit être valide',
+      ],
+      showPassword: false,
+      // object for message management
+      message: {
+        show: false,
+        content: '',
+        type: '',
+      },
+      /*
+       * For now LoggedUser is the current user logged info
+       * this will serve as temporary test and debug value
+       */
+      loggedUser: {},
+      // Object for form field binding
+      updateUserInfo: {
+        fname: '',
+        lname: '',
+        email: '',
+        password: '',
+      },
+    };
+  },
+  beforeMount() {
+    // set Bearer token in header of the future request
+    window.axios.defaults.headers.common = { Authorization: `Bearer ${localStorage.getItem('Authorization-token')}` };
```

Comment:
this will be done in #134 globally, keep it here for now (maybe add a comment)

Reply:
OK I will add a comment